CMake/FindSSL.cmake

```diff
@@ -0,0 +1,26 @@
+# - Try to find the Open ssl library (ssl)
```

Comment:
We don't allow finding libraries in the system.

Reply:
OK, I'll remove this file. Actually It's not used by contrib/libhdfs3-cmake, but used if you compile from contrib/libhdfs3 directly.

Reply:
Done

Reply:
@alexey-milovidov ，would you check again?